src/app/loadout/plug-drawer/SelectablePlug.tsx

```diff
@@ -96,6 +97,12 @@ function SelectablePlugDetails({
             ))}
           </div>
         )}
+        {plugDescriptions.communityInsight && (
+          <ClarityDescriptions
+            perk={plugDescriptions.communityInsight}
+            className={styles.clarityDescription}
+          />
+        )}
```

Comment:
This is more of the problem of how DIM is not utilizing available space 
And considering this is where people will pick what to use for load out it seems like important information to show not to mention we are adding aspects, fragments, and other things to
![image](https://user-images.githubusercontent.com/77716865/201519514-b54c4192-2a50-448f-ab04-c4af1e976c99.png)

Reply:
Maybe even make the container wider by like 20%
And I would like to make that description shorter because it is quite long just not sure how

Reply:
Few more options for how it could look with is the width of 300px instead of 250px
![image](https://user-images.githubusercontent.com/77716865/201520497-b0e7880a-8731-4466-b4c8-6ec2c8fdd511.png)

Reply:
That's fair, though the tiles still end up pretty tall even with those adjustments. I'm not totally opposed to this change, just think it would be good to get others thoughts. The plug picker might benefit from a larger redesign to accommodate this information.